### DISCUSSION TITLE
Home Depot - 2 Pack of v3's for $35.

### DISCUSSION THREAD

**Discussion by skilau** [gtxaspec/wz_mini_hacks#414] [General]
Hi all,
I forgot to post this a couple days ago, so the deal is probably nearly over, but Home Depot has a 2 pack of the v3's for $35.
(ie, $17.50 per v3 cam!)

https://www.homedepot.com/p/WYZE-Cam-v3-Wired-Home-Security-Camera-with-3-Months-Cam-Plus-Included-2-Pack-WYZEC3CP3X2/322233890

The deal I believe is good until your local store runs out, and then they yank the deal.

I picked up a couple more sets, mostly for "backup", in case my outdoor cams (in Minnesota) eventually die.

**Comment by alej908** [comment]
My Local Home Depot ran out of these, but Bestbuy still had some in stock. They price matched with just showing the sale at home depot on my phone!
